helpdesk_fieldservice/views/helpdesk_ticket_views.xml

```diff
@@ -45,4 +45,22 @@
             </field>
         </field>
     </record>
+
+    <menuitem id="menu_helpdesk_op"
+              name="Operations"
+              parent="helpdesk.menu_helpdesk_root"
+              sequence="10"/>
+
+    <menuitem id="menu_helpdesk_product"
+              name="Service Orders"
+              action="fieldservice.action_fsm_dash_order"
+              parent="menu_helpdesk_op"
+              sequence="20"/>
+
+    <data noupdate="1"><delete model="ir.ui.menu" id="helpdesk.helpdesk_ticket_menu_main"/></data>
+
+    <menuitem id="helpdesk_ticket_menu_main" 
+        action="helpdesk.helpdesk_ticket_action_main_tree"
+        sequence="10"
+        parent="menu_helpdesk_op"/>
```

Comment:
@osi-scampbell You don't need to delete the menu item to re-create it. Just change the parent with:

```
<menuitem id="helpdesk.helpdesk_ticket_menu_main" 
￼	        parent="menu_helpdesk_op"/>
```